#!/usr/bin/env python
PACKAGE = "lowlevel_ctrl"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Define parameters
gen.add("erpm_gain", double_t, 0, "ERPM Gain", 1200.0, 0.0, 5000.0)
gen.add("manual_vel", double_t, 0, "manual_vel",    0.0, -2,   3.0)
gen.add("manual_steer", double_t, 0, "manual_steer",    0.0, -0.43,   0.43)
gen.add("manual_ctrl",   bool_t,   0, "manual_ctrl",  False)

# Add more parameters as needed

# Generate the configuration
exit(gen.generate(PACKAGE, "lowlevel_ctrl", "LowLevelControl"))